### ISSUE TITLE
estimation of metric distance from the depth map using a regression model 

### ISSUE THREAD

**Issue by hosseinv** [nianticlabs/monodepth2#460]
Hi,
I am a newbie in te area of depth estimation using deep learning, so I apologize if my question seems naive.

I want to use the existing monodepth2 "pre-trained model" on KITTI to estimate the absolute distance. In this sense, I aim to do the inference on my "own dataset" (captured by my "GoPro camera") using this pre-trained model.

I am aware of the previous discussions here on the methods for generating absolute distance from the relative distance produced by the monodepth2 model.

However, I am wondering if it is possible to simply generate an absolute distance map from the relative distance map (produced using the pre-trained model KITTI) by solving a regression model by including the ground truth data (the absolute distance for several pixels on the depth map).

Something like (or other order of regression models):

AbsoluteDistance= A*(ReltiveDistance)+B

Thank you!

**Comment by daniyar-niantic** [comment]
Hi @hosseinv ,

The approach sounds reasonable, but there is no guarantee that the final depth would be metric or accurate.
